service/controller/legacy/v30/adapter/guest_load_balancers.go

```diff
@@ -111,6 +113,14 @@ func (a *GuestLoadBalancersAdapter) Adapt(cfg Config) error {
 	}
 	a.IngressElbScheme = externalELBScheme
 
+	// Ingress internal load balancer settings.
+	ingressInternalElbName, err := key.InternalLoadBalancerName(cfg.CustomObject.Spec.Cluster.Kubernetes.IngressController.Domain, cfg.CustomObject)
```

Comment:
For some reason the names we generate here are too long for e.g. our E2E sometimes.
Not entirely certain why this happens or if it's just an E2E problem.
![Screenshot from 2019-09-18 20-08-12](https://user-images.githubusercontent.com/9303709/65174111-89769680-da50-11e9-95eb-81990b1512ea.png)

Reply:
Crap
Maybe it needs to be shortened to IC
That’s anyway going to be removed with clusterapi

Reply:
There is only one per CF stack, right? 
Can we just name it `ingress-<clusterID>-internal`?

Reply:
That should be fine, yes

Reply:
I don't get this. Is there some crazy stuff on CI only? LB name for internal ingress is `ingress-<cluster_id>-internal`, which is <32

Reply:
Yes, I noticed this as well. Checking. I think the issue is that the clusterID in ci is way longer.

I will fix this, no need to worry for you.